app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py

```diff
@@ -0,0 +1,26 @@
+from jsk_rosbag_tools.bag_to_audio import bag_to_audio
```

Comment:
can you add try exception for this import?
if importerror, you can exit from this file to import.

Reply:
In that case, we dont need to update .ci.rosinstall

Reply:
Thank you for your advice.

I reverted ci.rosinstall update in [`634c1d3` (#54)](https://github.com/knorth55/app_manager_utils/pull/54/commits/634c1d38dc2c1b88ca8c2a6a163c23fc8bb6652d)

I added try-except and exit code in [`3a0f786` (#54)](https://github.com/knorth55/app_manager_utils/pull/54/commits/3a0f786cf40c0b750035d9513a79c2bc1d968036)